openpype/modules/deadline/repository/custom/plugins/GlobalJobPreLoad.py

```diff
@@ -359,11 +359,11 @@ def inject_openpype_environment(deadlinePlugin):
 
         args_str = subprocess.list2cmdline(args)
         print(">>> Executing: {} {}".format(exe, args_str))
-        process = ProcessUtils.SpawnProcess(
-            exe, args_str, os.path.dirname(exe)
+        process_exitcode = deadlinePlugin.RunProcess(
```

Comment:
Really, only this?

Reply:
I verified again, and just changing this solves our problem. I can't see the difference between the two.

Reply:
Ok, we will test on linux and let you know.